selfdrive/locationd/lagd.py

```diff
@@ -173,6 +173,9 @@ def __init__(self, CP: car.CarParams, dt: float,
     self.max_lat_accel = max_lat_accel
     self.max_lat_accel_diff = max_lat_accel_diff
 
+    # TODO: remove me, lagd is in shadow mode on release
+    self.is_release = Params().get_bool("IsReleaseBranch")
```

Comment:
I would pass it from the outside (with some more generic name `publish_estimate` / `use_estimate` / `..enabled`, so it's consistent with initial lag param restoration.

Reply:
I would think you want it to publish some estimate, or else there's no point running it at all in release